src/Fetch.js

```diff
@@ -478,7 +478,7 @@ function emscripten_start_fetch(fetch, successcb, errorcb, progresscb) {
 #if FETCH_DEBUG
     console.log('fetch: operation success. e: ' + e);
 #endif
-    if (onsuccess && Runtime.dynCall) Runtime.dynCall('vi', onsuccess, [fetch]);
+    if (onsuccess && Runtime.dynCall) Runtime.dynCall_vi(onsuccess, fetch);
```

Comment:
instead of `Runtime.dynCall_vi` this (and the rest) should be `Module['dynCall_vi']`

(do we also copy them to `Runtime`? I don't think so, and looking in the code I don't see that either)